examples/language-modeling/main.py

```diff
@@ -372,7 +372,7 @@
                 autoround.save_quantized(eval_folder, format=gpu_format, use_triton=False, inplace=inplace)
             elif "auto_awq" in gpu_format:  # pragma: no cover
                 eval_folder = f'{export_dir}-awq'
-                autoround.save_quantized(eval_folder, format=gpu_format, inplace=inplace, model_path=model_name)
+                autoround.save_quantized(eval_folder, format=gpu_format, inplace=inplace)
 
         if 'xpu' in deployment_device:
```

Comment:
line 370  elif "gptq" in gpu_format:
                eval_folder = f'{export_dir}-gpu' ##changed to gptq